packages/backend-defaults/src/entrypoints/cache/CacheManager.ts

```diff
@@ -89,22 +95,81 @@ export class CacheManager {
       }
     }
 
+    // Read store-specific options from config
+    const storeOptions = CacheManager.parseStoreOptions(store, config, logger);
+
     return new CacheManager(
       store,
       connectionString,
       options.onError,
       logger,
       defaultTtl,
+      storeOptions,
     );
   }
 
+  /**
+   * Parse store-specific options from configuration.
+   *
+   * @param store - The cache store type ('redis', 'memcache', or 'memory')
+   * @param config - The configuration service
+   * @param logger - Optional logger for warnings
+   * @returns The parsed store options
+   */
+  private static parseStoreOptions(
+    store: string,
+    config: RootConfigService,
+    logger?: LoggerService,
+  ): CacheStoreOptions | undefined {
+    const storeConfigPath = `backend.cache.${store}`;
+
+    if (store === 'redis' && config.has(storeConfigPath)) {
+      return CacheManager.parseRedisOptions(storeConfigPath, config, logger);
+    }
+
+    return undefined;
+  }
+
+  /**
+   * Parse Redis-specific options from configuration.
+   */
+  private static parseRedisOptions(
+    storeConfigPath: string,
+    config: RootConfigService,
+    logger?: LoggerService,
+  ): RedisCacheStoreOptions {
+    const redisOptions: RedisCacheStoreOptions = {};
+    const redisConfig = config.getConfig(storeConfigPath);
+
+    redisOptions.client = redisConfig.getOptional('client');
+
+    if (redisConfig.has('cluster')) {
+      const clusterConfig = redisConfig.getConfig('cluster');
```

Comment:
Whenever you add config, the config.d.ts file in the root of the package needs to be updated accordingly. In particular, if some fields can be secrets in here, they should be marked with `@visibility secret` so that they don't leak.